packages/client/src/index.ts

```diff
@@ -41,7 +60,22 @@ export interface RoutingV1HttpApiClient {
    * Returns an async generator of PeerInfos that can provide the content
    * for the passed CID
    */
-  getProviders(cid: CID, options?: AbortOptions): AsyncGenerator<PeerInfo>
+  getProviders(cid: CID, options?: AbortOptions): AsyncGenerator<Record>
+
+  /**
+   * Returns an async generator of PeerInfos for the provided PeerId
+   */
+  getPeers(pid: PeerId, options?: AbortOptions): AsyncGenerator<Record>
```

Comment:
getPeerInfo? instead of getPeers?

Can the return type be guaranteed to be `PeerRecord`?

Reply:
Yes, here I can do that - https://specs.ipfs.tech/routing/http-routing-v1/#get-routing-v1-peers-peer-id.